build/ios/mojom/mojom_objc_generator.py

```diff
@@ -250,6 +250,22 @@ def CppToObjC(self, accessor):
             return d;
         }()""" % args
 
+class BaseValueMojoTypemap(MojoTypemap):
+    @staticmethod
+    def IsMojoType(kind):
+        return (mojom.IsStructKind(kind) and
+                kind.qualified_name == 'mojo_base.mojom.DictionaryValue')
```

Comment:
Actually, going to rename this to `BaseDictionaryValue` and add support for `mojo_base.mojom.ListValue` as well